scriptmodules/emulators/dosbox-staging.sh

```diff
@@ -56,15 +51,23 @@ function configure_dosbox-staging() {
 
     [[ "$md_id" == "remove" ]] && return
 
+    # copy required resources
+    local config_dir="$md_conf_root/pc"
+    cp -R "$md_build/contrib/resources/"{freedos-cpi,freedos-keyboard,glshaders,translations} "$config_dir/"
+    chown -R $user:$user "$config_dir"
```

Comment:
Change to
```
chown -R $user: "$config_dir"
```
otherwise will have to rebase after #3577 gets merged.

Reply:
Thanks for catching this. Script updated.